app/Http/Controllers/UploadController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class UploadController extends Controller
+{
+    public function upload()
+    {
+        return view ('upload');
+    }
+    public function proses_upload(Request $request){
+		$this->validate($request, [
+			'file' => 'required|image',
+		]);
+ 
+		$file = $request->file('file');
+ 
+      	        // nama file
+		echo 'File Name: '.$file->getClientOriginalName();
+		echo '<br> Uploaded! You can close this now..... just click outside the popup ';
+ 
+      	//         // ekstensi file
+		// echo 'File Extension: '.$file->getClientOriginalExtension();
+		// echo '<br>';
+ 
+      	//         // real path
+		// echo 'File Real Path: '.$file->getRealPath();
+		// echo '<br>';
+ 
+      	//         // ukuran file
+		// echo 'File Size: '.$file->getSize();
+		// echo '<br>';
+ 
+      	//         // tipe mime
+		// echo 'File Mime Type: '.$file->getMimeType();
+ 
+      	        // isi dengan nama folder tempat kemana file diupload
```

Comment:
Please remove comments like this in a PR, because you can actually rewind in git so you still have these codes in commit history